### PR TITLE
Expand storage tests to cover table and queue stats commands

### PR DESCRIPTION
**Pull request by troydai**

### CODE REVIEW

File: src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py [hunk 1]
```diff
@@ -488,29 +488,36 @@ def tear_down(self):
...
             self.resource_group, self.account))
         self.cmd('group delete --name {} --no-wait --yes'.format(self.resource_group))
+
+    @classmethod
+    def generate_account_name(cls):
+        return 'vcrstorage{}'.format(''.join(choice(digits) for _ in range(12)))
+
+    @classmethod
+    def generate_random_tag(cls):
+        return '_{}_'.format(''.join((choice(ascii_lowercase + digits) for _ in range(4))))
```
Comment (tjprescott) [inline src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py]: This essentially does for the storage account what the ResourceGroupVCRTestBase does for the resource group name?

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py [hunk 2]
```diff
@@ -813,3 +813,20 @@ def body(self):
...
+    def test_table_and_queue_stats(self):
+        # combine two tests to avoid repeatedly creating accounts.
+        self.execute()
+
+    def body(self):
+        self.cmd('storage table stats --account-name {}'.format(self.account),
+                 checks=JMESPathCheck('geoReplication.status', 'live'))
+        self.cmd('storage queue stats --account-name {}'.format(self.account),
+                 checks=JMESPathCheck('geoReplication.status', 'live'))
+        assert True
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py]: What is the purpose of this statement?

Comment (troydai) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py]: should be removed.

File: src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py (hunk 1 above)
Comment (troydai) [inline src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py]: I was just extracting this logic into its own method. Yes, the mechanism is same but the length of the random string is shorter here.

**Pull request merged**